ivadomed/training.py

```diff
@@ -165,7 +165,6 @@ def train(model_params, dataset_train, dataset_val, training_params, log_directo
         for i, batch in enumerate(train_loader):
             # GET SAMPLES
             if model_params["name"] == "HeMISUnet":
-                batch["input"] = batch["input"].squeeze() if batch["input"].shape[1] > 1 else batch["input"]
```

Comment:
I reverted an earlier commit that changed the shape of `DataLoader`'s `batch["input"]` for HeMIS; I mistakenly made this commit when using an improperly formatted dataset. With correctly formatted data, this line is superfluous and can actually cause errors, e.g. in #594 where the `filter_absent_mask` can cause `batch["input"]`'s length to be smaller than the batch size.

Apologies for this ugly revert-commit, I couldn't figure out how to do this neatly with a `git rebase` :sweat: